force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -355,15 +423,16 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of new Person objects.
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
-
+        List<Person> personList = new List<Person>();
         // Loop through each name in the provided list of names
+        for(String name: names){
             // Create a new Person object for each name
-            
+            Person person = new Person(name, age);
             // Add the new Person object to the list of Person objects
-        
-        
+            personList.add(person);
```

Comment:
Good job.  Youd don't need that variable in your loop though just add to the list directly